.circleci/config.yml

```diff
@@ -4,7 +4,8 @@ version: 2.1
 stdenv: &stdenv
   environment:
     GOCACHE: &gocache /tmp/go-build
-    IMAGE: &image saschagrunert/criocircle:1.1.0
+    IMAGE: &image quay.io/haircommander/cri-o-go-1.12:latest
+    IMAGELEGACY: &imagelegacy quay.io/haircommander/cri-o-go-1.10:latest
```

Comment:
We ought to have these images in a more generic account on quay.io.  Something like quay.io/cri-o/ or the like.  If you need to, you can set up a email list to act as the account owner, that's what we did with quay.io/buildah and quay.io/podman

Reply:
we have quay.io/crio (I also reserved quay.io/cri-o, but we already use crio), I think I'm gonna push these images there, and rename them as @saschagrunert suggested above :)